contributor-guides/for-staff.md

```diff
@@ -11,12 +11,11 @@ Here are some important things to figure out in the first couple of days:
 
 * [Using space guide](../activities/office-management/using-space.md) - how we use the spaces in the office
 * Email - you should have an account, make sure you can send and receive
-* Team Public Code chat - say hello, we're a welcoming bunch
+* [Team Public Code chat](../activities/communication/using-chat.md) - say hello, we're a welcoming bunch
```

Comment:
We use staff everywhere else. Also, we can't use 'Public Code' ever to talk about ourselves.

```suggestion
* [Staff chat](../activities/communication/using-chat.md) - say hello, we're a welcoming bunch
```